src/server/iframe.html.js

```diff
@@ -3,6 +3,11 @@ export default function (headHtml) {
     <!DOCTYPE html>
     <html>
       <head>
+        <script>
+          if (window.parent !== window) {
```

Comment:
@wyattdanger yes, we can do it with the `head.html`. As @rogchap said, this should be the default.